### PR TITLE
[Merged by Bors] - chore: adaptations for nightly-2024-07-09

### PR DESCRIPTION
**Pull request by jcommelin**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/GroupTheory/Congruence/Basic.lean
Comment (Ruben-VandeVelde) [inline Mathlib/GroupTheory/Congruence/Basic.lean:201]: Also could be flipped

File: Mathlib/LinearAlgebra/AffineSpace/AffineSubspace.lean
Comment (Ruben-VandeVelde) [inline Mathlib/LinearAlgebra/AffineSpace/AffineSubspace.lean:350]: Another one

File: Mathlib/NumberTheory/Bernoulli.lean
Comment (Ruben-VandeVelde) [inline Mathlib/NumberTheory/Bernoulli.lean:344]: We have the special syntax for adaptation notes now  (and I think the linter would have complained if not for the sneaky underscore)

File: Mathlib/RingTheory/DedekindDomain/Factorization.lean
Comment (Ruben-VandeVelde) [inline Mathlib/RingTheory/DedekindDomain/Factorization.lean:76]: .

**Review by Ruben-VandeVelde — APPROVED** [review]
Two comments I'd like you to fix before merging (Mathlib/CategoryTheory/Bicategory/Coherence.lean and Mathlib/NumberTheory/Bernoulli.lean).

I think we should try to keep the divergence between master and the bump branches minimal, so I'd like to see use make `ext_iff` `protected` on master, and fix the few lemmas that are currently in the wrong direction. That shouldn't block this PR, though, and I can try to make some time to work on that myself as well.

File: Mathlib/Data/Fin/Basic.lean
Comment (semorrison) [reply Mathlib/Data/Fin/Basic.lean:846]: Indeed, done.

File: Mathlib/Data/Finmap.lean
Comment (semorrison) [reply Mathlib/Data/Finmap.lean:167]: Ditto. :-)

File: Mathlib/Data/Sigma/Lex.lean
Comment (semorrison) [reply Mathlib/Data/Sigma/Lex.lean:119]: Done.

File: Mathlib/GroupTheory/Congruence/Basic.lean
Comment (semorrison) [reply Mathlib/GroupTheory/Congruence/Basic.lean:201]: Again unsure what you intended here.

File: Mathlib/LinearAlgebra/AffineSpace/AffineSubspace.lean
Comment (semorrison) [reply Mathlib/LinearAlgebra/AffineSpace/AffineSubspace.lean:350]: As above, not sure what change on master you had in mind.

File: Mathlib/NumberTheory/Bernoulli.lean
Comment (semorrison) [reply Mathlib/NumberTheory/Bernoulli.lean:344]: Fixed.

File: Mathlib/RingTheory/DedekindDomain/Factorization.lean
Comment (semorrison) [reply Mathlib/RingTheory/DedekindDomain/Factorization.lean:76]: Backported.

**Comment by semorrison** [comment]
bors merge

**Comment by semorrison** [comment]
bors p=10

**Pull request closed without merging**
